TestApps/Sasquatch/Sasquatch/ViewControllers/MSAnalyticsViewController.swift

```diff
@@ -1,18 +1,23 @@
 import UIKit
 
-private var kPropertiesSection: Int = 3
+private var kEventPropertiesSection: Int = 2
+private var kTargetPropertiesSection: Int = 3
```

Comment:
This is a section _index_, no?  Just a naming nit :)

Reply:
Sure, will update